src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -112,7 +112,7 @@ function validate_identifier(type, action, input_options, is_options_from_file)
  */
 function validate_no_extra_options(type, action, input_options, is_options_from_file) {
     let valid_options; // for performance, we use Set as data structure
-    const from_file_condition = (type === TYPES.ACCOUNT || type === TYPES.BUCKET) &&
+    const from_file_condition = (type === TYPES.ACCOUNT || type === TYPES.BUCKET || type === TYPES.CONNECTION) &&
```

Comment:
Why did you add a change here?
The `--from-file` flag seems not related to the connection.

Reply:
No, I want the from_file for adding a connection.